### ISSUE TITLE
Problem with upload files

### ISSUE THREAD

**Issue by KingaJelen** [pytrip/pytrip#536]
![image](https://user-images.githubusercontent.com/33397810/83307185-65389f80-a204-11ea-85e0-8842a0f544c3.png)

**Comment by grzanka** [comment]
It looks that in this particular case user tried to load DDD files from a directory which contained also other files (PDF and SPC).

**Comment by nbassler** [comment]
Yes, this is not a pytrip issue, since the same error would persist in TRiP98.
user should do "mydddpath/*.ddd"

**Comment by grzanka** [comment]
The question is: can we assume that pytripGUI (!) users will be aware of the convention to store only DDD files in a directory from where they should be loaded ?

What we could do is to add filtering based on file extension (ddd and DDD), as suggested by TRiP98 documentation:
http://bio.gsi.de/DOCS/TRiP98/PRO/DOCS/trip98cmdddd.html
![image](https://user-images.githubusercontent.com/7374733/83418580-21bc7c00-a424-11ea-86b9-4515a806274c.png)

In following place:
https://github.com/pytrip/pytrip/blob/v3.1.2/pytrip/tripexecuter/plan.py#L385
the `ddd` command is used with a pure directory as an argument.

I'd add here a bit of logic: first a check if `self.default_kernel.ddd_path` is a directory and not an expression like `/a/b/c/*.xxx`. If it is a directory I'd then use following code to load DDD files:

```
        output.append('ddd "{:s}.*ddd" / read'.format(self.default_kernel.ddd_path))
        output.append('ddd "{:s}.*DDD" / read'.format(self.default_kernel.ddd_path))
```

this would cover most cases of loading DDD files via GUI where only directories can be selected.

**Comment by grzanka** [comment]
This filtering can be also done on the level of GUI

**Comment by nbassler** [comment]
I see, but I am not too happy about too much magic handling based on missing file extensions. TRiP itself also does that, and I find it leads to more confusion.

Instead of doing magic, I find it better to give clear error messages. Maybe a small function which checks if a given ddd file looks like a *ddd file? Run it on every file (glob), complain when user tries to add a file.

It is a bit design-philosophy. PyTRiP could add an "extra service" for the user, to complain when user feeds it with bad data. Or: PyTRiPGUI had the purpose to give less tech-savvy users a chance to work with TRiP, so probably there could be some checks on GUI level instead.
